tests/tests/constants/testContracts.ts

```diff
@@ -1,4 +1,33 @@
 import { AbiItem } from "web3-utils";
+import solc from "solc";
+
+export function compileSolidity(contractContent: string, contractName: string = "Test") {
```

Comment:
Yeah, this is awesome! We should do all of our solidity-based tests this way.

Reply:
So at first I thought to myself that the repo would be cleaner if we did this for every single contract but then I thought that recompiling every contract every test might slightly increase the duration of testing. Thoughts @crystalin @JoshOrndorff ?